flake-module.nix

```diff
@@ -6,7 +6,7 @@ common:
   imports = [
     common.inputs.flake-root.flakeModule
     common.inputs.haskell-flake.flakeModule
-    common.inputs.nix-health.flakeModule
+    common.inputs.omnix-flake.flakeModules.default
```

Comment:
Just delete this module, we are not using it since https://github.com/nammayatri/nammayatri/pull/9213